module/Finna/src/Finna/RecordDriver/SolrForward.php

```diff
@@ -580,65 +621,97 @@ public function getPlayingTimes()
     }
 
     /**
-     * Get all presenters
+     * Get presenters as an assoc array
      *
      * @return array
      */
-    public function getAllPresenters()
```

Comment:
Tämä ja muut metodien nimien muutokset ovat taaksepäin yhteensopimattomia. Hajoaako Kavin näkymä vai riittääkö formatterin muutos?

Reply:
Tästä kävin kurkkailemassa kavia ja näyttäisi siltä, ettei näitä kyseenalaisia funktioita käytetä missään. Ja tämä getPresenters on käytössä vain formatterissa näillä näkymin.